pkg/quota/v1/evaluator/core/resource_claims.go

```diff
@@ -0,0 +1,165 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package core
+
+import (
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	resourceapi "k8s.io/api/resource/v1alpha3"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apiserver/pkg/admission"
+	quota "k8s.io/apiserver/pkg/quota/v1"
+	"k8s.io/apiserver/pkg/quota/v1/generic"
+	resourceinternal "k8s.io/kubernetes/pkg/apis/resource"
+	resourceversioned "k8s.io/kubernetes/pkg/apis/resource/v1alpha3"
+)
+
+// the name used for object count quota
+var claimObjectCountName = generic.ObjectCountQuotaResourceNameFor(resourceapi.SchemeGroupVersion.WithResource("resourceclaims").GroupResource())
```

Comment:
Actually, this isn't the alias. It's the standard `count/resourceclaims.resource.k8s.io`.

When I tried to remove this, I found that because this evaluator registers itself for the `resourceclaims.resource.k8s.io` ResourceGroup, the normal code no longer handled that count. Therefore I am going to keep this.